src/applications/ivc-champva/10-7959a/config/form.js

```diff
@@ -103,12 +110,61 @@ const formConfig = {
       title: 'Sponsor information',
       pages: {
         page2: {
-          path: 'sponsor-information',
+          path: 'sponsor-info',
           title: 'Name',
           ...sponsorNameSchema,
         },
       },
     },
+    beneficiaryInformation: {
+      title: 'Beneficiary information',
+      pages: {
+        page2a: {
+          path: 'beneficiary-info',
+          title: 'Beneficiary information',
+          ...applicantNameDobSchema,
+        },
+        page2b: {
+          path: 'beneficiary-identification-info',
+          title: formData => `${fnp(formData)} identification information`,
+          ...applicantMemberNumberSchema,
+        },
+        page2c: {
+          path: 'beneficiary-address',
+          title: formData => `${fnp(formData)} address`,
+          // Only show if we have addresses to pull from:
+          depends: formData =>
+            get('certifierRole', formData) !== 'applicant' &&
+            get('street', formData?.certifierAddress),
+          CustomPage: props => {
+            const extraProps = {
+              ...props,
+              customTitle: `${fnp(props.data)} address`,
+              customDescription:
+                'We’ll send any important information about this form to this address.',
+              customSelectText:
+                'Does the beneficiary have the same address as you?',
+              positivePrefix: 'Yes, their address is',
+              negativePrefix: 'No, they have a different address',
+            };
+            return ApplicantAddressCopyPage(extraProps);
```

Comment:
Address component is shared with form 10-10d, so set up some custom text variables so the view matches what we need in this form.